pkgs/tools/misc/betterdiscordctl/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "betterdiscordctl";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "bb010g";
+    repo = "betterdiscordctl";
+    rev = "v${version}";
+    sha256 = "0qpmm5l8jhm7k0kqblc0bnr9fl4b6z8iddhjar03bb4kqgr962fa";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/bb010g/betterdiscordctl/pull/67/commits/f1c7170fc2626d9aec4d244977b5a73c401aa1d4.patch";
+      sha256 = "003zqd9ljb9h674sjwjvvdfs7q4cw0p1ydg3lax132vb4vz9k0zi";
+    })
+  ];
+
+  preBuild = ''
+    substituteInPlace betterdiscordctl \
+      --replace "DISABLE_UPGRADE=" "DISABLE_UPGRADE=1" \
+      --replace "nix=" "nix=yes"
```

Comment:
Please use `yes` instead of `1` for consistency.


Also, substituting `nix=` with `nix=yes` will affect the following line as well:
```diff
@@ -207,7 +207,7 @@ while :; do
       die_non_empty '--flatpak'
       ;;
     --nix)
-      nix=yes
+      nix=yesyes
       ;;
     --upgrade-url)
       if [[ ${2+x} ]]; then upgrade_url=$2; shift
```
It doesn't particularly matter in this case but I'd like to avoid it if possible.

Reply:
Mhm, I'm not sure if that's possible, sadly.

Reply:
Are regular expressions not supported? Using `^nix=$` should fix it.

Reply:
Doesn't look like it, unfortunately